src/data/gitmojis.json

```
@@ -235,7 +235,7 @@
       "emoji": "🚚",
       "entity": "&#1F69A;",
       "code": ":truck:",
-      "description": "Move or rename files.",
+      "description": "Move or rename resources (files, paths, routes).",
```

```suggestion
      "description": "Move or rename resources (e.g.: files, paths, routes).",
```

@KaKi87 @johannchopin what d'you think? that way it's a little bit more explicity my intention of it being exemples. The way I see, this could be applied to variables also since they are resources anyway :D

---

I think it's clear enough without the `e.g` 🤔

---

I like the idea of adding `e.g.`.

---

> I think it's clear enough without the e.g thinking

well, @johannchopin, if it's clear enough without the `e.g.`, it's even clearer with it, so if we keep the `e.g.` then we are all happy :tada: :sweat_smile: does this work for you?